Frontend/ui-customer/src/Components/SearchLocationModel/View.purs

```diff
@@ -241,19 +239,21 @@ sourceDestinationEditTextView state push =
       [ height WRAP_CONTENT
       , width MATCH_PARENT
       , orientation HORIZONTAL
+      , background "#303440"
+      , cornerRadius 4.0 
+      , stroke if state.isSource == Just true && state.isSearchLocation == LocateOnMap then "1,#FDD836" else "0,#FDD836"
       ][ editText
-            [ height $ V 45
+            [ height $ V 37
             , weight 1.0
             , text state.source
-            , color Color.black800
-            , stroke if state.isSource == Just true && state.isSearchLocation == LocateOnMap then "1,#FDD836" else "0,#FDD836"
-            , background Color.white900
+            , color if not(state.isSource == Just true && state.isSearchLocation /= LocateOnMap) then "#A7A7A7" else Color.white900
+            , background "#303440"
+            , stroke "0,#000000"
```

Comment:
@abhigyanshukla97 If stroke is '0' you can remove

Reply:
In EditText there is a blue underline to remove it we have to use this else it wont work.